query/upgrade_test.go

```diff
@@ -0,0 +1,62 @@
+//go:build upgrade
+
+/*
+ * Copyright 2023 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package query
+
+import (
+	"testing"
+	"time"
+
+	"github.com/dgraph-io/dgraph/dgraphtest"
+	"github.com/dgraph-io/dgraph/x"
+)
+
+func TestMain(m *testing.M) {
+	conf := dgraphtest.NewClusterConfig().WithNumAlphas(3).WithNumZeros(3).WithReplicas(3).
+		WithACL(20 * time.Second).WithEncryption().WithVersion("0c9f60156")
```

Comment:
what SHA does this refer to?

Reply:
Probably one before the `v23.0.0-beta1` but this will get replaced in the next PR. This PR ensures that we can run upgrade tests in query package, any upgrade tests.